packages/web/src/components/nav/desktop/NavAudio.tsx

```diff
@@ -27,6 +30,29 @@ const messages = {
   claimRewards: 'Claim Rewards'
 }
 
+const validRewardIds: Set<ChallengeRewardID> = new Set([
+  'track-upload',
+  'referrals',
+  'ref-v',
+  'mobile-install',
+  'connect-verified',
+  'listen-streak',
+  'profile-completion',
+  'referred',
+  'send-first-tip'
+])
+
+/** Pulls rewards from remoteconfig */
+const useActiveRewardIds = () => {
+  const rewardsString = useRemoteVar(StringKeys.CHALLENGE_REWARD_IDS)
+  if (rewardsString === null) return []
+  const activeRewards = rewardsString.split(',') as ChallengeRewardID[]
```

Comment:
Total nit, but was meaning this is rewards, and then filteredRewards is activeRewards!